sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2967,7 +2967,7 @@ object functions {
    * See [[java.text.SimpleDateFormat]] for valid date and time format patterns
```

Comment:
If you don't mind, I would like to say here that currently supported pattern for seconds fractions is `SSS` only. The second s fractions can be parsed but will be ignored while casting to TimestampType.